libraries/botbuilder-dialogs-adaptive/src/generators/languageGeneratorManager.ts

```diff
@@ -23,55 +23,64 @@ export class LanguageGeneratorManager {
     /// <summary>
     /// multi language lg resources. en -> [resourcelist].
     /// </summary>
-    private _multiLanguageResources: Map<string, IResource[]>;
+    private _multiLanguageResources: Map<string, Resource[]>;
 
     public constructor(resourceManager: ResourceExplorer) {
         this._resourceExporer = resourceManager;
-
+        this._resourceExporer.changed = this.resourceExplorerChanged.bind(this);
```

Comment:
FYI - If you write the resourceExplorerChanged function as a lamba, then you don't need to bind(this) in order for callbacks to work.

const resourceExplorerChanged = async(event: ResourceChangeEvent, resources: Resource[]): Promise<void> => { ...}

Reply:
Cool, thanks.